README.md

```diff
@@ -58,3 +138,37 @@ GeoServer will then be available at:
 ```
 http://localhost:9090
 ```
+
+## Running using Tomcat and IntelliJ
+
+Install Tomcat 8.5
+
+Install Java version 11
+
+Set the maven profile to `tomcat`.
+
+![img.png](img.png)
```

Comment:
this screenshot is not valid

Reply:
Hi @craigrose  the binary `*.jar` files in the `lib` directory, were they intentionally added to the repo or can be excluded in the `.gitignore` file?

Reply:
The binary files are needed there for now because the required extension version is not currently available in a maven repo.

Reply:
Found the right maven repo. Binaries are now removed.